src/__test__/calculate.test.js

```diff
@@ -0,0 +1,58 @@
+import operate from '../logic/operate';
+import calculate from '../logic/calculate';
+
+it('returns the addition of two numbers', () => {
+  const result = operate(5, 8, '+');
+  expect(result).toBe('13');
+});
+
+it('returns the addition of two numbers as not equal', () => {
+  const result = operate(5, 8, '+');
+  expect(result).not.toBe('10');
+});
+
+it('returns the subtraction of two numbers', () => {
+  const result = operate(8, 5, '-');
+  expect(result).toBe('3');
+});
+
+it('returns the subtraction of two numbers', () => {
```

Comment:
- Am guessing this is the negative test case but you are describing it just like the positive one. This is inconsistent with the first negative test. I expect it to be `returns the subtraction of two numbers as not equal`. Please make the required changes for all test cases following this one.